### DISCUSSION TITLE
Disable power reporting for TS011F_plug_1

### DISCUSSION THREAD

**Discussion by nkrepo** [Koenkk/zigbee2mqtt#20794] [Devices]
Hi all,
I have a lot of TS011F_plug_1 smart plugs and the zigbee network is flooded with useless power reports (and probably inaccurate ).
I looking if there is a way to disable power reporting for those devices. 

Ι tried to disable the reporting from the reporting tab.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/8989693/0972e3e9-09f3-41fa-9f22-c8f6e3fbea19)
No luck.

I tried to firmware upgrade them in order to become the "dumper" version TS011F_plug_3 with no luck either unfortunately.

Is there another way ?

**Comment by fips21** [comment]
Normally there shouldn't be any setting under Reporting tab. Did you create them manually? Did you already try to remove a device and join again to see if it's gone afterwards?

![Zwischenablage01](https://github.com/Koenkk/zigbee2mqtt/assets/8333194/9713576d-ff8d-4b21-b780-834fca812b91)

**Reply by nkrepo** [reply]
No i did not create them manually, I will try removing and re-joining the device.
